pages/battle.php

```diff
@@ -171,12 +177,18 @@ function processBattleFightEnd(BattleManager|BattleManagerV2 $battle, User $play
         if ($player->daily_tasks->hasTaskType(DailyTask::ACTIVITY_PVP)) {
             $player->daily_tasks->progressTask(DailyTask::ACTIVITY_PVP, 1);
         }
+        // Loot
+        $player->system->db->query("UPDATE `loot` SET `user_id` = {$player->user_id}, `battle_id` = NULL WHERE `battle_id` = {$player->battle_id}");
+        if ($player->system->db->last_num_rows > 0) {
+            $result .= "You have claimed the loot being carried by your opponent.[br]";
+        }
 
     } else if ($battle->isOpponentWinner()) {
         $result .= "You lose. You were taken back to your village by some allied ninja.[br]";
         $player->pvp_losses++;
         $player->last_pvp_ms = System::currentTimeMs();
         $player->last_death_ms = System::currentTimeMs();
+        $player->pvp_immunity_ms = System::currentTimeMs() + (5 * 60 * 1000); // 5 minutes
```

Comment:
Optionally we could put this value as a constant in a class somewhere, for easier configuration

Reply:
Any preference on where it would belong? `User` probably.